content/docs/reference/config/policy-and-telemetry/metrics.md

```diff
@@ -0,0 +1,98 @@
+---
+title: Default Metrics
+description: Default Metrics exported from Istio through Mixer.
+weight: 50
+---
+
+This page presents details about the metrics that Istio collects when using its initial configuration. You can add and remove metrics by changing configuration at any time, but this is the built-in set. They can be found
+[here](https://github.com/istio/istio/blob/{{<branch_name>}}/install/kubernetes/helm/istio/charts/mixer/templates/config.yaml)
+under the section with “kind: metric”. It uses [metric
+template](/docs/reference/config/policy-and-telemetry/templates/metric/) to define these metrics.
+
+We will describe metrics first and then the labels for each metric.
+
+## Metrics
+
+*   **Request Count**: This is a `COUNTER`
+    [metric](https://github.com/istio/istio/blob/{{<branch_name>}}/install/kubernetes/templates/istio-mixer.yaml.tmpl#L786:9)
+    incremented for every request handled by an Istio proxy.
+
+*   **Request Duration**: This is a `DISTRIBUTION`
+    [metric](https://github.com/istio/istio/blob/{{<branch_name>}}/install/kubernetes/templates/istio-mixer.yaml.tmpl#L802:9)
+    which measures the duration of the request (as observed by the server-side
+    proxy). This metric is obtained from envoy proxy.
```

Comment:
Looking at this again, I think we should just drop `This metric is obtained from envoy proxy` from all the descriptions. I think this is more confusing than necessary, as envoy has its own metrics for clusters that differ from these metrics.

Reply:
Thanks..Fixed